browser-test/login.test.js

```diff
@@ -15,7 +15,7 @@ describe("Selenium tests for <Login />", () => {
   beforeAll(async () => {
     await initializeData();
     driver = await getDriver(Builder, new firefox.Options().headless());
-  });
+  }, 30000);
```

Comment:
Timeout of 30 seconds only for `beforeAll` hook. But if the driver gets initialized in 5 seconds it will not wait for 30 seconds. i.e. async callback is resolved.